packages/jupyter-ai/jupyter_ai/models.py

```diff
@@ -87,13 +87,34 @@ class ClearMessage(BaseModel):
     type: Literal["clear"] = "clear"
 
 
+class PendingMessage(BaseModel):
+    type: Literal["pending"] = "pending"
+    id: str
+    time: float
+    body: str
+    persona: Persona
+    ellipsis: bool = True
```

Comment:
I already have a use for this in my custom ChatHandlers. Using it as more of a temporary message rather than a loading indication. If you don't think that it is worth an extra field, I am fine with that. It is just a minor formatting config. What do you think?

Reply:
No worries, we can keep it here then! 👍